doc/api/stream.md

```diff
@@ -1011,94 +1010,92 @@ However, this would be better implemented as a [Transform][] stream. See
 // alternative example below under the Transform section.
 
 const Readable = require('stream').Readable;
-const util = require('util');
 
-util.inherits(SimpleProtocol, Readable);
+class SimpleProtocol extends Readable {
+  constructor(source, options) {
+    super(options);
+    if (!(this instanceof SimpleProtocol))
+      return new SimpleProtocol(source, options);
```

Comment:
These checks were there in the original example in case someone called `SimpleProtocol()` instead of `new SimpleProtocol()`. Calling without `new` is not possible using classes, so yes, the `if` is not useful.